### ISSUE TITLE
Unable to download GOG games

### ISSUE THREAD

**Issue by Nico0340** [Heroic-Games-Launcher/HeroicGamesLauncher#2333]
### Describe the bug

Unable to download any game from GOG library, see the attached file.
it's seems the app didn't find the link to download.

[https://i.postimg.cc/w9n5TnjT/gog.png](url)

### Add logs

```text
(11:56:37) INFO:    [Connection]:       Connectivity: check-online
(11:56:37) INFO:    [Connection]:       Pinging external endpoints
(11:56:37) INFO:    [Legendary]:        Running command: /tmp/.mount_HeroicSwVLVV/resources/app.asar.unpacked/build/bin/linux/legendary --version
(11:56:37) INFO:    [Legendary]:        Legendary location: /tmp/.mount_HeroicSwVLVV/resources/app.asar.unpacked/build/bin/linux/legendary
(11:56:37) INFO:    [Gog]:              GOGDL location: /tmp/.mount_HeroicSwVLVV/resources/app.asar.unpacked/build/bin/linux/gogdl
(11:56:37) WARNING: [Backend]:          Failed to register protocol with OS.
(11:56:37) INFO:    [Connection]:       Connectivity: online
(11:56:37) INFO:    [Gog]:              Running command: /tmp/.mount_HeroicSwVLVV/resources/app.asar.unpacked/build/bin/linux/gogdl --version
(11:56:38) INFO:    [Backend]:          AreWeAntiCheatYet data downloaded
(11:56:38) INFO:    [Backend]:          

Heroic Version: 2.5.2 Yamato
Legendary Version:  0.20.31 Dark Energy (hotfix #5)
GOGdl Version: 
OS: Debian GNU/Linux KERNEL: 5.10.0-19-amd64 ARCH: x64
CPU: Intel Core™ i7-4790K @4 GOVERNOR: schedutil
RAM: Total: 15.58 GiB Available: 13.35 GiB
GRAPHICS: GPU0: NVIDIA GeForce GTX 970 VRAM: 4096MB DRIVER: 525.60.13 
PROTOCOL: x11

(11:56:38) INFO:    [Backend]:          Checking for new Heroic Updates
(11:56:38) INFO:    [Backend]:          Checking for current version changelog
(11:56:38) INFO:    [Frontend]:         Updated to a new version, cleaaning up the cache.
(11:56:38) INFO:    [Legendary]:        Running command: /tmp/.mount_HeroicSwVLVV/resources/app.asar.unpacked/build/bin/linux/legendary cleanup
(11:56:38) INFO:    [Frontend]:         Refreshing Library
(11:56:38) INFO:    [Legendary]:        Refreshing library...
(11:56:38) INFO:    [Legendary]:        Refreshing Epic Games...
(11:56:38) INFO:    [Legendary]:        Game list updated, got 149 games & DLCs
(11:56:38) INFO:    [Backend]:          Frontend Ready
(11:56:38) INFO:    [Frontend]:         No cache found, getting data from legendary...
(11:56:38) INFO:    [Legendary]:        Refreshing library...
(11:56:38) INFO:    [Legendary]:        Game list updated, got 149 games & DLCs
(11:56:38) INFO:    [Frontend]:         Force Update
(11:56:38) INFO:    [Legendary]:        Running command: /tmp/.mount_HeroicSwVLVV/resources/app.asar.unpacked/build/bin/linux/legendary list --third-party
(11:56:39) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:39) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:39) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:39) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:56:39) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:39) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:56:39) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:39) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:39) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:39) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:39) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:39) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:56:39) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:39) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:56:39) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:39) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:39) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:56:39) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:56:39) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:39) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:39) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:39) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:39) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:56:39) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:39) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:39) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:39) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:56:39) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:39) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:56:39) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:56:39) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:39) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:39) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:39) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:39) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:39) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:39) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:39) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:56:39) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:39) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:56:39) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:39) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:56:39) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:39) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:56:39) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:39) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:39) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:39) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:39) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:39) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:56:39) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:56:39) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:39) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:39) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Winetricks]:       Downloading Winetricks
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [Legendary]:        Getting Info from Epic API
(11:56:40) INFO:    [DXVKInstaller]:    Updating vkd3d to: vkd3d-proton-2.7
(11:56:40) INFO:    [DXVKInstaller]:    Updating dxvk to: dxvk-2.0
(11:56:41) INFO:    [DXVKInstaller]:    downloaded dxvk
(11:56:41) INFO:    [DXVKInstaller]:    extracting dxvk
(11:56:41) INFO:    [DXVKInstaller]:    downloaded vkd3d
(11:56:41) INFO:    [DXVKInstaller]:    extracting vkd3d
(11:56:41) INFO:    [DXVKInstaller]:    vkd3d updated!
(11:56:41) INFO:    [DXVKInstaller]:    dxvk updated!
(11:56:43) INFO:    [Backend]:          Starting the Download Queue
(11:58:12) INFO:    [Gog]:              Logging using GOG credentials
(11:58:12) INFO:    [Gog]:              Login Successful
(11:58:12) INFO:    [Gog]:              Getting data about the user
(11:58:13) INFO:    [Gog]:              Saved user data to config
(11:58:13) INFO:    [Frontend]:         Refreshing Library
(11:58:13) INFO:    [Gog]:              Getting GOG library
(11:58:13) INFO:    [Gog]:              Number of library pages: 2
(11:58:13) INFO:    [Gog]:              Getting data for page 2
(11:58:15) WARNING: [Gog]:              Unable to get covers from gamesdb for Cyberpunk 2077 Goodies Collection. Trying to get it from api.gog.com
(11:58:15) WARNING: [Gog]:              Couldn't get info from api.gog.com, Using fallback vertical image
(11:58:23) INFO:    [Backend]:          Changing Language to: fr
(11:58:24) INFO:    [Gog]:              Saved games data
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:58:25) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:58:25) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:58:25) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:58:25) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:58:25) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:58:25) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:25) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:58:26) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:26) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:58:26) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:26) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:58:26) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:26) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:58:26) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:26) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:26) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:26) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:58:26) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:26) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:26) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:26) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:26) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:58:26) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:58:26) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:58:26) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:58:26) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:58:26) INFO:    [Legendary]:        Getting Info from Epic API
(11:58:26) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:58:26) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:58:26) ERROR:   [Legendary]:        Error Getting Info from Epic API
(11:58:34) INFO:    [Backend]:          Getting HowLongToBeat data for Timberborn
(11:58:34) INFO:    [Gog]:              Getting game metadata: /tmp/.mount_HeroicSwVLVV/resources/app.asar.unpacked/build/bin/linux/gogdl info 1711765822 --token <redacted> --lang=en-US --os windows
(11:58:34) WARNING: [Gog]:              Failed to get Install Info for 1711765822 using Windows as platform, returning empty object
(11:58:34) INFO:    [Gog]:              Getting game metadata: /tmp/.mount_HeroicSwVLVV/resources/app.asar.unpacked/build/bin/linux/gogdl info 1711765822 --token <redacted> --lang=en-US --os windows
(11:58:34) WARNING: [Gog]:              Failed to get Install Info for 1711765822 using Windows as platform, returning empty object
(11:58:44) INFO:    [Gog]:              Getting game metadata: /tmp/.mount_HeroicSwVLVV/resources/app.asar.unpacked/build/bin/linux/gogdl info 1711765822 --token <redacted> --lang=en-US --os windows
(11:58:45) DEBUG:   [Backend]:          {"free":69452955648,"diskSize":158084792320,"message":"64.68 GiB / 147.23 GiB","validPath":true}
(11:58:45) WARNING: [Gog]:              Failed to get Install Info for 1711765822 using Windows as platform, returning empty object
(11:58:58) INFO:    [Gog]:              Getting game metadata: /tmp/.mount_HeroicSwVLVV/resources/app.asar.unpacked/build/bin/linux/gogdl info 1711765822 --token <redacted> --lang=en-US --os windows
(11:58:58) DEBUG:   [Backend]:          {"free":69452959744,"diskSize":158084792320,"message":"64.68 GiB / 147.23 GiB","validPath":true}
(11:58:58) WARNING: [Gog]:              Failed to get Install Info for 1711765822 using Windows as platform, returning empty object
```

### Steps to reproduce

1. Go to Library
2. Search for (exemple) Timberborn
3. Clic to install
4. See error (nothing happend)

### Expected behavior

he should download the game

### Screenshots

[https://i.postimg.cc/w9n5TnjT/gog.png](url)

### Heroic Version

Latest Stable

### System Information

OS: Debian GNU/Linux 11 (bullseye) x86_ 
Host: MS-7919 1.0 
Kernel: 5.10.0-19-amd64 
Shell: bash 5.1.4 
Resolution: 1920x1080, 1920x1080 
DE: Xfce 4.16 
Terminal: xfce4-terminal 
CPU: Intel i7-4790K (8) @ 4.400GHz 
GPU: NVIDIA GeForce GTX 970 
Memory: 2110MiB / 15953MiB

**Comment by flavioislima** [comment]
Debian is not supported. I believe you have an issue with your connection that is somehow blocking Heroic requests.
Open a Discussion here on Github or go to our discord. Check the list of supported distros on our ReadMe.

**Comment by Nico0340** [comment]
it was working pretty well on 2.41 before upgrading to the latest version.
i don't change anything with my connection ; i can go to the epic store or gog store with the heroic.
but thx so much for you anwser and you time.

**Comment by imLinguin** [comment]
Although Debian is not supported it may be fixed in the next 2.6.0 release. Stay tuned
